src/app/collection/shared/included/collection-learning-object-card/collection-learning-object-card.component.html

```diff
@@ -0,0 +1,28 @@
+<div class = "container">
+
+
+    <div class = "header" [ngClass] = "learnObj.length">
+            <div class = "overlay">
+                {{learnObj.length}}
+            </div>
+    </div>
+
+    
+    <div class="obj-info">
+         <div class = "obj-title">
+            {{learnObj.name}}
+        </div>
+        <div>
+            {{learnObj.date | date:'longDate'}}
+        </div>
+        
```

Comment:
Could we remove the extra lines of space in this file? like 2,10, and 18. Just helps clean up the markup